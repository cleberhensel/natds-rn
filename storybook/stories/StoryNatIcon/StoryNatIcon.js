import React, { useState } from 'react';
import {
  SafeAreaView,
  FlatList,
  View,
  ScrollView,
  Dimensions,
} from 'react-native';
import {
  NatContainer,
  NatText,
  NatIcon,
  NatTextInput,
  ThemeNaturaLight,
} from '../../../lib';

const styles = {
  defaultScreen: {
    flex: 1,
    width: Dimensions.get('window').width,
  },
};

export const StoryNatIcon = () => {
  const [input, setInput] = useState('');
  const iconList = [
    { name: 'filled-action-add' },
    { name: 'filled-action-cancel' },
    { name: 'filled-action-check' },
    { name: 'filled-action-rating' },
    { name: 'filled-action-subtract' },
    { name: 'filled-brand-naturarosacea' },
    { name: 'filled-media-pause' },
    { name: 'filled-media-play' },
    { name: 'filled-navigation-directionleft' },
    { name: 'filled-navigation-directionright' },
    { name: 'filled-navigation-directiontbottom' },
    { name: 'filled-navigation-directiontop' },
    { name: 'filled-navigation-menu' },
    { name: 'filled-navigation-more' },
    { name: 'outlined-action-add' },
    { name: 'outlined-action-attachment' },
    { name: 'outlined-action-autofilter' },
    { name: 'outlined-action-calendar' },
    { name: 'outlined-action-cancel' },
    { name: 'outlined-action-change' },
    { name: 'outlined-action-check' },
    { name: 'outlined-action-closedbox' },
    { name: 'outlined-action-copy' },
    { name: 'outlined-action-cycle' },
    { name: 'outlined-action-delete' },
    { name: 'outlined-action-done' },
    { name: 'outlined-action-download' },
    { name: 'outlined-action-edit' },
    { name: 'outlined-action-favorite' },
    { name: 'outlined-action-filter' },
    { name: 'outlined-action-gridsquare' },
    { name: 'outlined-action-help' },
    { name: 'outlined-action-history' },
    { name: 'outlined-action-like' },
    { name: 'outlined-action-list' },
    { name: 'outlined-action-love' },
    { name: 'outlined-action-mic' },
    { name: 'outlined-action-micrec' },
    { name: 'outlined-action-newrequest' },
    { name: 'outlined-action-order' },
    { name: 'outlined-action-phone' },
    { name: 'outlined-action-print' },
    { name: 'outlined-action-rating' },
    { name: 'outlined-action-request' },
    { name: 'outlined-action-save' },
    { name: 'outlined-action-schedule' },
    { name: 'outlined-action-search' },
    { name: 'outlined-action-selectedalllines' },
    { name: 'outlined-action-send' },
    { name: 'outlined-action-settings' },
    { name: 'outlined-action-share' },
    { name: 'outlined-action-subtract' },
    { name: 'outlined-action-update' },
    { name: 'outlined-action-upload' },
    { name: 'outlined-action-visibility' },
    { name: 'outlined-action-visibilityoff' },
    { name: 'outlined-alert-empty' },
    { name: 'outlined-alert-indeterminate' },
    { name: 'outlined-alert-info' },
    { name: 'outlined-communication-chat' },
    { name: 'outlined-communication-email' },
    { name: 'outlined-communication-help' },
    { name: 'outlined-communication-message' },
    { name: 'outlined-communication-occurrence' },
    { name: 'outlined-communication-readEmail' },
    { name: 'outlined-communication-signal' },
    { name: 'outlined-communication-speaker' },
    { name: 'outlined-communication-treatment' },
    { name: 'outlined-content-adddocument' },
    { name: 'outlined-content-agility' },
    { name: 'outlined-content-book' },
    { name: 'outlined-content-brightness' },
    { name: 'outlined-content-changeview' },
    { name: 'outlined-content-channel' },
    { name: 'outlined-content-christmascard' },
    { name: 'outlined-content-consistency' },
    { name: 'outlined-content-divulgation' },
    { name: 'outlined-content-education' },
    { name: 'outlined-content-gift' },
    { name: 'outlined-content-global' },
    { name: 'outlined-content-health' },
    { name: 'outlined-content-lock' },
    { name: 'outlined-content-magic' },
    { name: 'outlined-content-medal' },
    { name: 'outlined-content-menu' },
    { name: 'outlined-content-mirror' },
    { name: 'outlined-content-produtivity' },
    { name: 'outlined-content-store' },
    { name: 'outlined-content-wifi' },
    { name: 'outlined-finance-bag' },
    { name: 'outlined-finance-cards' },
    { name: 'outlined-finance-charging' },
    { name: 'outlined-finance-creditcard' },
    { name: 'outlined-finance-debit' },
    { name: 'outlined-finance-discount' },
    { name: 'outlined-finance-littlemachine' },
    { name: 'outlined-finance-money' },
    { name: 'outlined-finance-moneypaper' },
    { name: 'outlined-finance-promotion' },
    { name: 'outlined-finance-shoppingcart' },
    { name: 'outlined-finance-shoppingcartproduct' },
    { name: 'outlined-finance-tagdiscount' },
    { name: 'outlined-finance-transfermoney' },
    { name: 'outlined-media-app' },
    { name: 'outlined-media-camera' },
    { name: 'outlined-media-monitor' },
    { name: 'outlined-media-play' },
    { name: 'outlined-navigation-arrowbottom' },
    { name: 'outlined-navigation-arrowleft' },
    { name: 'outlined-navigation-arrowright' },
    { name: 'outlined-navigation-arrowtop' },
    { name: 'outlined-navigation-close' },
    { name: 'outlined-navigation-directionbottom' },
    { name: 'outlined-navigation-directionleft' },
    { name: 'outlined-navigation-directionright' },
    { name: 'outlined-navigation-directiontop' },
    { name: 'outlined-navigation-exit' },
    { name: 'outlined-navigation-goback' },
    { name: 'outlined-navigation-gofront' },
    { name: 'outlined-navigation-home' },
    { name: 'outlined-navigation-menu' },
    { name: 'outlined-place-bus' },
    { name: 'outlined-place-forklift' },
    { name: 'outlined-place-local' },
    { name: 'outlined-place-truck' },
    { name: 'outlined-place-waze' },
    { name: 'outlined-product-bagproduct' },
    { name: 'outlined-product-beard' },
    { name: 'outlined-product-brandsproduct' },
    { name: 'outlined-product-childish' },
    { name: 'outlined-product-dailycare' },
    { name: 'outlined-product-hair' },
    { name: 'outlined-product-makeup' },
    { name: 'outlined-product-outlet' },
    { name: 'outlined-product-perfumery' },
    { name: 'outlined-product-promotionproduct' },
    { name: 'outlined-product-vegan' },
    { name: 'outlined-social-addcontact' },
    { name: 'outlined-social-birthday' },
    { name: 'outlined-social-collaborate' },
    { name: 'outlined-social-contact' },
    { name: 'outlined-social-digitalconsultant' },
    { name: 'outlined-social-groupofpeople' },
    { name: 'outlined-social-myprofile' },
    { name: 'outlined-social-network' },
    { name: 'outlined-social-person' },
  ];

  const renderItem = name => (
    <View style={{ flex: 1, flexDirection: 'row' }}>
      <NatIcon name={name} fill={ThemeNaturaLight.palette.text.primary} />
      <NatText type="body1" value={name} />
    </View>
  );

  const filterIconList = filter => iconList.filter(item => item.name.includes(filter.toLowerCase()));

  return (
    <SafeAreaView style={styles.defaultScreen}>
      <NatContainer>
        <ScrollView>
          <NatText type="h6" value="ICONS SVG" />
          <NatTextInput
            label=""
            placeholder="Search"
            onChangeText={setInput}
            value={input}
            size="normal"
            type="default"
            icon={null}
          />
          <FlatList
            data={filterIconList(input)}
            renderItem={({ item }) => renderItem(item.name)}
            keyExtractor={item => item.name}
          />
        </ScrollView>
      </NatContainer>
    </SafeAreaView>
  );
};

StoryNatIcon.propTypes = {
};
