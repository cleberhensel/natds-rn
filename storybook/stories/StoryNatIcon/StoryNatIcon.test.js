import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components/native';

import { StoryNatIcon } from './StoryNatIcon';

describe('StoryNatIcon component', () => {
  it('Should render StoryNatIcon component ', () => {
    const button = renderer.create(
      <StoryNatIcon />,
    ).toJSON();

    expect(button).toMatchSnapshot('StoryNatIcon snapshot');
  });
});
