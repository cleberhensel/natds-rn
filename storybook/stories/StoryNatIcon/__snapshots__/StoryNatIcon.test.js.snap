// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StoryNatIcon component Should render StoryNatIcon component  1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
>
  <View
    style={
      Array [
        Object {
<<<<<<< HEAD
          "backgroundColor": "#FFFFFF",
=======
<<<<<<< HEAD
          "backgroundColor": "#FFFFFF",
=======
          "backgroundColor": "white",
>>>>>>> origin/update/react
>>>>>>> origin/update/react
          "bottom": 0,
          "flexBasis": 0,
          "flexGrow": 1,
          "flexShrink": 1,
          "left": 0,
          "paddingBottom": 0,
          "paddingLeft": 16,
          "paddingRight": 16,
          "paddingTop": 0,
          "right": 0,
          "top": 0,
        },
        Object {
          "backgroundColor": "#FFF",
          "width": 750,
        },
      ]
    }
  >
    <RCTScrollView>
      <View>
        <View
          margin={24}
          style={
            Array [
              Object {
<<<<<<< HEAD
=======
                "flexBasis": 0,
                "flexGrow": 1,
                "flexShrink": 1,
>>>>>>> origin/update/react
                "marginBottom": 24,
                "marginLeft": 0,
                "marginRight": 0,
                "marginTop": 0,
                "paddingBottom": 0,
                "paddingLeft": 0,
                "paddingRight": 0,
                "paddingTop": 0,
              },
            ]
          }
        >
          <Text
            color="#333333"
            style={
              Array [
                Object {
<<<<<<< HEAD
=======
                  "backgroundColor": "#FFF",
>>>>>>> origin/update/react
                  "color": "#333333",
                  "fontSize": 20,
                  "fontWeight": "500",
                  "lineHeight": 30,
                },
              ]
            }
            type="h6"
          >
            ICONS SVG
          </Text>
        </View>
        <RCTScrollView
          data={
            Array [
              Object {
                "name": "filled-action-add",
              },
              Object {
                "name": "filled-action-cancel",
              },
              Object {
                "name": "filled-action-check",
              },
              Object {
                "name": "filled-action-rating",
              },
              Object {
                "name": "filled-action-subtract",
              },
              Object {
                "name": "filled-brand-naturarosacea",
              },
              Object {
                "name": "filled-media-pause",
              },
              Object {
                "name": "filled-media-play",
              },
              Object {
                "name": "filled-navigation-directionleft",
              },
              Object {
                "name": "filled-navigation-directionright",
              },
              Object {
                "name": "filled-navigation-directiontbottom",
              },
              Object {
                "name": "filled-navigation-directiontop",
              },
              Object {
                "name": "filled-navigation-menu",
              },
              Object {
                "name": "filled-navigation-more",
              },
              Object {
                "name": "outlined-action-add",
              },
              Object {
                "name": "outlined-action-attachment",
              },
              Object {
                "name": "outlined-action-autofilter",
              },
              Object {
                "name": "outlined-action-calendar",
              },
              Object {
                "name": "outlined-action-cancel",
              },
              Object {
                "name": "outlined-action-change",
              },
              Object {
                "name": "outlined-action-check",
              },
              Object {
                "name": "outlined-action-closedbox",
              },
              Object {
                "name": "outlined-action-copy",
              },
              Object {
                "name": "outlined-action-cycle",
              },
              Object {
                "name": "outlined-action-delete",
              },
              Object {
                "name": "outlined-action-done",
              },
              Object {
                "name": "outlined-action-download",
              },
              Object {
                "name": "outlined-action-edit",
              },
              Object {
                "name": "outlined-action-favorite",
              },
              Object {
                "name": "outlined-action-filter",
              },
              Object {
                "name": "outlined-action-gridsquare",
              },
              Object {
                "name": "outlined-action-help",
              },
              Object {
                "name": "outlined-action-history",
              },
              Object {
                "name": "outlined-action-like",
              },
              Object {
                "name": "outlined-action-list",
              },
              Object {
                "name": "outlined-action-love",
              },
              Object {
                "name": "outlined-action-mic",
              },
              Object {
                "name": "outlined-action-micrec",
              },
              Object {
                "name": "outlined-action-newrequest",
              },
              Object {
                "name": "outlined-action-order",
              },
              Object {
                "name": "outlined-action-phone",
              },
              Object {
                "name": "outlined-action-print",
              },
              Object {
                "name": "outlined-action-rating",
              },
              Object {
                "name": "outlined-action-request",
              },
              Object {
                "name": "outlined-action-save",
              },
              Object {
                "name": "outlined-action-schedule",
              },
              Object {
                "name": "outlined-action-search",
              },
              Object {
                "name": "outlined-action-selectedalllines",
              },
              Object {
                "name": "outlined-action-send",
              },
              Object {
                "name": "outlined-action-settings",
              },
              Object {
                "name": "outlined-action-share",
              },
              Object {
                "name": "outlined-action-subtract",
              },
              Object {
                "name": "outlined-action-update",
              },
              Object {
                "name": "outlined-action-upload",
              },
              Object {
                "name": "outlined-action-visibility",
              },
              Object {
                "name": "outlined-action-visibilityoff",
              },
              Object {
                "name": "outlined-alert-empty",
              },
              Object {
                "name": "outlined-alert-indeterminate",
              },
              Object {
                "name": "outlined-alert-info",
              },
              Object {
                "name": "outlined-communication-chat",
              },
              Object {
                "name": "outlined-communication-email",
              },
              Object {
                "name": "outlined-communication-help",
              },
              Object {
                "name": "outlined-communication-message",
              },
              Object {
                "name": "outlined-communication-occurrence",
              },
              Object {
                "name": "outlined-communication-readEmail",
              },
              Object {
                "name": "outlined-communication-signal",
              },
              Object {
                "name": "outlined-communication-speaker",
              },
              Object {
                "name": "outlined-communication-treatment",
              },
              Object {
                "name": "outlined-content-adddocument",
              },
              Object {
                "name": "outlined-content-agility",
              },
              Object {
                "name": "outlined-content-book",
              },
              Object {
                "name": "outlined-content-brightness",
              },
              Object {
                "name": "outlined-content-changeview",
              },
              Object {
                "name": "outlined-content-channel",
              },
              Object {
                "name": "outlined-content-christmascard",
              },
              Object {
                "name": "outlined-content-consistency",
              },
              Object {
                "name": "outlined-content-divulgation",
              },
              Object {
                "name": "outlined-content-education",
              },
              Object {
                "name": "outlined-content-gift",
              },
              Object {
                "name": "outlined-content-global",
              },
              Object {
                "name": "outlined-content-health",
              },
              Object {
                "name": "outlined-content-lock",
              },
              Object {
                "name": "outlined-content-magic",
              },
              Object {
                "name": "outlined-content-medal",
              },
              Object {
                "name": "outlined-content-menu",
              },
              Object {
                "name": "outlined-content-mirror",
              },
              Object {
                "name": "outlined-content-produtivity",
              },
              Object {
                "name": "outlined-content-store",
              },
              Object {
                "name": "outlined-content-wifi",
              },
              Object {
                "name": "outlined-finance-bag",
              },
              Object {
                "name": "outlined-finance-cards",
              },
              Object {
                "name": "outlined-finance-charging",
              },
              Object {
                "name": "outlined-finance-creditcard",
              },
              Object {
                "name": "outlined-finance-debit",
              },
              Object {
                "name": "outlined-finance-discount",
              },
              Object {
                "name": "outlined-finance-littlemachine",
              },
              Object {
                "name": "outlined-finance-money",
              },
              Object {
                "name": "outlined-finance-moneypaper",
              },
              Object {
                "name": "outlined-finance-promotion",
              },
              Object {
                "name": "outlined-finance-shoppingcart",
              },
              Object {
                "name": "outlined-finance-shoppingcartproduct",
              },
              Object {
                "name": "outlined-finance-tagdiscount",
              },
              Object {
                "name": "outlined-finance-transfermoney",
              },
              Object {
                "name": "outlined-media-app",
              },
              Object {
                "name": "outlined-media-camera",
              },
              Object {
                "name": "outlined-media-monitor",
              },
              Object {
                "name": "outlined-media-play",
              },
              Object {
                "name": "outlined-navigation-arrowbottom",
              },
              Object {
                "name": "outlined-navigation-arrowleft",
              },
              Object {
                "name": "outlined-navigation-arrowright",
              },
              Object {
                "name": "outlined-navigation-arrowtop",
              },
              Object {
                "name": "outlined-navigation-close",
              },
              Object {
                "name": "outlined-navigation-directionbottom",
              },
              Object {
                "name": "outlined-navigation-directionleft",
              },
              Object {
                "name": "outlined-navigation-directionright",
              },
              Object {
                "name": "outlined-navigation-directiontop",
              },
              Object {
                "name": "outlined-navigation-exit",
              },
              Object {
                "name": "outlined-navigation-goback",
              },
              Object {
                "name": "outlined-navigation-gofront",
              },
              Object {
                "name": "outlined-navigation-home",
              },
              Object {
                "name": "outlined-navigation-menu",
              },
              Object {
                "name": "outlined-place-bus",
              },
              Object {
                "name": "outlined-place-forklift",
              },
              Object {
                "name": "outlined-place-local",
              },
              Object {
                "name": "outlined-place-truck",
              },
              Object {
                "name": "outlined-place-waze",
              },
              Object {
                "name": "outlined-product-bagproduct",
              },
              Object {
                "name": "outlined-product-beard",
              },
              Object {
                "name": "outlined-product-brandsproduct",
              },
              Object {
                "name": "outlined-product-childish",
              },
              Object {
                "name": "outlined-product-dailycare",
              },
              Object {
                "name": "outlined-product-hair",
              },
              Object {
                "name": "outlined-product-makeup",
              },
              Object {
                "name": "outlined-product-outlet",
              },
              Object {
                "name": "outlined-product-perfumery",
              },
              Object {
                "name": "outlined-product-promotionproduct",
              },
              Object {
                "name": "outlined-product-vegan",
              },
              Object {
                "name": "outlined-social-addcontact",
              },
              Object {
                "name": "outlined-social-birthday",
              },
              Object {
                "name": "outlined-social-collaborate",
              },
              Object {
                "name": "outlined-social-contact",
              },
              Object {
                "name": "outlined-social-digitalconsultant",
              },
              Object {
                "name": "outlined-social-groupofpeople",
              },
              Object {
                "name": "outlined-social-myprofile",
              },
              Object {
                "name": "outlined-social-network",
              },
              Object {
                "name": "outlined-social-person",
              },
            ]
          }
          disableVirtualization={false}
          getItem={[Function]}
          getItemCount={[Function]}
          horizontal={false}
          initialNumToRender={10}
          keyExtractor={[Function]}
          maxToRenderPerBatch={10}
          numColumns={1}
          onContentSizeChange={[Function]}
          onEndReachedThreshold={2}
          onLayout={[Function]}
          onMomentumScrollEnd={[Function]}
          onScroll={[Function]}
          onScrollBeginDrag={[Function]}
          onScrollEndDrag={[Function]}
          removeClippedSubviews={false}
          renderItem={[Function]}
          scrollEventThrottle={50}
          stickyHeaderIndices={Array []}
          updateCellsBatchingPeriod={50}
          viewabilityConfigCallbackPairs={Array []}
          windowSize={21}
        >
          <View>
            <View
              onLayout={[Function]}
              style={null}
            >
              <View
                style={
                  Object {
                    "flex": 1,
                    "flexDirection": "row",
                  }
                }
              >
                <SvgMock
                  height={24}
                  style={
                    Object {
                      "fill": "#333333",
                    }
                  }
                  width={24}
                />
                <View
                  margin={24}
                  style={
                    Array [
                      Object {
<<<<<<< HEAD
=======
                        "flexBasis": 0,
                        "flexGrow": 1,
                        "flexShrink": 1,
>>>>>>> origin/update/react
                        "marginBottom": 24,
                        "marginLeft": 0,
                        "marginRight": 0,
                        "marginTop": 0,
                        "paddingBottom": 0,
                        "paddingLeft": 0,
                        "paddingRight": 0,
                        "paddingTop": 0,
                      },
                    ]
                  }
                >
                  <Text
                    color="#333333"
                    style={
                      Array [
                        Object {
<<<<<<< HEAD
=======
                          "backgroundColor": "#FFF",
>>>>>>> origin/update/react
                          "color": "#333333",
                          "fontSize": 16,
                          "fontWeight": "400",
                          "lineHeight": 24,
                        },
                      ]
                    }
                    type="body1"
                  >
                    filled-action-add
                  </Text>
                </View>
              </View>
            </View>
            <View
              onLayout={[Function]}
              style={null}
            >
              <View
                style={
                  Object {
                    "flex": 1,
                    "flexDirection": "row",
                  }
                }
              >
                <SvgMock
                  height={24}
                  style={
                    Object {
                      "fill": "#333333",
                    }
                  }
                  width={24}
                />
                <View
                  margin={24}
                  style={
                    Array [
                      Object {
<<<<<<< HEAD
=======
                        "flexBasis": 0,
                        "flexGrow": 1,
                        "flexShrink": 1,
>>>>>>> origin/update/react
                        "marginBottom": 24,
                        "marginLeft": 0,
                        "marginRight": 0,
                        "marginTop": 0,
                        "paddingBottom": 0,
                        "paddingLeft": 0,
                        "paddingRight": 0,
                        "paddingTop": 0,
                      },
                    ]
                  }
                >
                  <Text
                    color="#333333"
                    style={
                      Array [
                        Object {
<<<<<<< HEAD
=======
                          "backgroundColor": "#FFF",
>>>>>>> origin/update/react
                          "color": "#333333",
                          "fontSize": 16,
                          "fontWeight": "400",
                          "lineHeight": 24,
                        },
                      ]
                    }
                    type="body1"
                  >
                    filled-action-cancel
                  </Text>
                </View>
              </View>
            </View>
            <View
              onLayout={[Function]}
              style={null}
            >
              <View
                style={
                  Object {
                    "flex": 1,
                    "flexDirection": "row",
                  }
                }
              >
                <SvgMock
                  height={24}
                  style={
                    Object {
                      "fill": "#333333",
                    }
                  }
                  width={24}
                />
                <View
                  margin={24}
                  style={
                    Array [
                      Object {
<<<<<<< HEAD
=======
                        "flexBasis": 0,
                        "flexGrow": 1,
                        "flexShrink": 1,
>>>>>>> origin/update/react
                        "marginBottom": 24,
                        "marginLeft": 0,
                        "marginRight": 0,
                        "marginTop": 0,
                        "paddingBottom": 0,
                        "paddingLeft": 0,
                        "paddingRight": 0,
                        "paddingTop": 0,
                      },
                    ]
                  }
                >
                  <Text
                    color="#333333"
                    style={
                      Array [
                        Object {
<<<<<<< HEAD
=======
                          "backgroundColor": "#FFF",
>>>>>>> origin/update/react
                          "color": "#333333",
                          "fontSize": 16,
                          "fontWeight": "400",
                          "lineHeight": 24,
                        },
                      ]
                    }
                    type="body1"
                  >
                    filled-action-check
                  </Text>
                </View>
              </View>
            </View>
            <View
              onLayout={[Function]}
              style={null}
            >
              <View
                style={
                  Object {
                    "flex": 1,
                    "flexDirection": "row",
                  }
                }
              >
                <SvgMock
                  height={24}
                  style={
                    Object {
                      "fill": "#333333",
                    }
                  }
                  width={24}
                />
                <View
                  margin={24}
                  style={
                    Array [
                      Object {
<<<<<<< HEAD
=======
                        "flexBasis": 0,
                        "flexGrow": 1,
                        "flexShrink": 1,
>>>>>>> origin/update/react
                        "marginBottom": 24,
                        "marginLeft": 0,
                        "marginRight": 0,
                        "marginTop": 0,
                        "paddingBottom": 0,
                        "paddingLeft": 0,
                        "paddingRight": 0,
                        "paddingTop": 0,
                      },
                    ]
                  }
                >
                  <Text
                    color="#333333"
                    style={
                      Array [
                        Object {
<<<<<<< HEAD
=======
                          "backgroundColor": "#FFF",
>>>>>>> origin/update/react
                          "color": "#333333",
                          "fontSize": 16,
                          "fontWeight": "400",
                          "lineHeight": 24,
                        },
                      ]
                    }
                    type="body1"
                  >
                    filled-action-rating
                  </Text>
                </View>
              </View>
            </View>
            <View
              onLayout={[Function]}
              style={null}
            >
              <View
                style={
                  Object {
                    "flex": 1,
                    "flexDirection": "row",
                  }
                }
              >
                <SvgMock
                  height={24}
                  style={
                    Object {
                      "fill": "#333333",
                    }
                  }
                  width={24}
                />
                <View
                  margin={24}
                  style={
                    Array [
                      Object {
<<<<<<< HEAD
=======
                        "flexBasis": 0,
                        "flexGrow": 1,
                        "flexShrink": 1,
>>>>>>> origin/update/react
                        "marginBottom": 24,
                        "marginLeft": 0,
                        "marginRight": 0,
                        "marginTop": 0,
                        "paddingBottom": 0,
                        "paddingLeft": 0,
                        "paddingRight": 0,
                        "paddingTop": 0,
                      },
                    ]
                  }
                >
                  <Text
                    color="#333333"
                    style={
                      Array [
                        Object {
<<<<<<< HEAD
=======
                          "backgroundColor": "#FFF",
>>>>>>> origin/update/react
                          "color": "#333333",
                          "fontSize": 16,
                          "fontWeight": "400",
                          "lineHeight": 24,
                        },
                      ]
                    }
                    type="body1"
                  >
                    filled-action-subtract
                  </Text>
                </View>
              </View>
            </View>
            <View
              onLayout={[Function]}
              style={null}
            >
              <View
                style={
                  Object {
                    "flex": 1,
                    "flexDirection": "row",
                  }
                }
              >
                <SvgMock
                  height={24}
                  style={
                    Object {
                      "fill": "#333333",
                    }
                  }
                  width={24}
                />
                <View
                  margin={24}
                  style={
                    Array [
                      Object {
<<<<<<< HEAD
=======
                        "flexBasis": 0,
                        "flexGrow": 1,
                        "flexShrink": 1,
>>>>>>> origin/update/react
                        "marginBottom": 24,
                        "marginLeft": 0,
                        "marginRight": 0,
                        "marginTop": 0,
                        "paddingBottom": 0,
                        "paddingLeft": 0,
                        "paddingRight": 0,
                        "paddingTop": 0,
                      },
                    ]
                  }
                >
                  <Text
                    color="#333333"
                    style={
                      Array [
                        Object {
<<<<<<< HEAD
=======
                          "backgroundColor": "#FFF",
>>>>>>> origin/update/react
                          "color": "#333333",
                          "fontSize": 16,
                          "fontWeight": "400",
                          "lineHeight": 24,
                        },
                      ]
                    }
                    type="body1"
                  >
                    filled-brand-naturarosacea
                  </Text>
                </View>
              </View>
            </View>
            <View
              onLayout={[Function]}
              style={null}
            >
              <View
                style={
                  Object {
                    "flex": 1,
                    "flexDirection": "row",
                  }
                }
              >
                <SvgMock
                  height={24}
                  style={
                    Object {
                      "fill": "#333333",
                    }
                  }
                  width={24}
                />
                <View
                  margin={24}
                  style={
                    Array [
                      Object {
<<<<<<< HEAD
=======
                        "flexBasis": 0,
                        "flexGrow": 1,
                        "flexShrink": 1,
>>>>>>> origin/update/react
                        "marginBottom": 24,
                        "marginLeft": 0,
                        "marginRight": 0,
                        "marginTop": 0,
                        "paddingBottom": 0,
                        "paddingLeft": 0,
                        "paddingRight": 0,
                        "paddingTop": 0,
                      },
                    ]
                  }
                >
                  <Text
                    color="#333333"
                    style={
                      Array [
                        Object {
<<<<<<< HEAD
=======
                          "backgroundColor": "#FFF",
>>>>>>> origin/update/react
                          "color": "#333333",
                          "fontSize": 16,
                          "fontWeight": "400",
                          "lineHeight": 24,
                        },
                      ]
                    }
                    type="body1"
                  >
                    filled-media-pause
                  </Text>
                </View>
              </View>
            </View>
            <View
              onLayout={[Function]}
              style={null}
            >
              <View
                style={
                  Object {
                    "flex": 1,
                    "flexDirection": "row",
                  }
                }
              >
                <SvgMock
                  height={24}
                  style={
                    Object {
                      "fill": "#333333",
                    }
                  }
                  width={24}
                />
                <View
                  margin={24}
                  style={
                    Array [
                      Object {
<<<<<<< HEAD
=======
                        "flexBasis": 0,
                        "flexGrow": 1,
                        "flexShrink": 1,
>>>>>>> origin/update/react
                        "marginBottom": 24,
                        "marginLeft": 0,
                        "marginRight": 0,
                        "marginTop": 0,
                        "paddingBottom": 0,
                        "paddingLeft": 0,
                        "paddingRight": 0,
                        "paddingTop": 0,
                      },
                    ]
                  }
                >
                  <Text
                    color="#333333"
                    style={
                      Array [
                        Object {
<<<<<<< HEAD
=======
                          "backgroundColor": "#FFF",
>>>>>>> origin/update/react
                          "color": "#333333",
                          "fontSize": 16,
                          "fontWeight": "400",
                          "lineHeight": 24,
                        },
                      ]
                    }
                    type="body1"
                  >
                    filled-media-play
                  </Text>
                </View>
              </View>
            </View>
            <View
              onLayout={[Function]}
              style={null}
            >
              <View
                style={
                  Object {
                    "flex": 1,
                    "flexDirection": "row",
                  }
                }
              >
                <SvgMock
                  height={24}
                  style={
                    Object {
                      "fill": "#333333",
                    }
                  }
                  width={24}
                />
                <View
                  margin={24}
                  style={
                    Array [
                      Object {
<<<<<<< HEAD
=======
                        "flexBasis": 0,
                        "flexGrow": 1,
                        "flexShrink": 1,
>>>>>>> origin/update/react
                        "marginBottom": 24,
                        "marginLeft": 0,
                        "marginRight": 0,
                        "marginTop": 0,
                        "paddingBottom": 0,
                        "paddingLeft": 0,
                        "paddingRight": 0,
                        "paddingTop": 0,
                      },
                    ]
                  }
                >
                  <Text
                    color="#333333"
                    style={
                      Array [
                        Object {
<<<<<<< HEAD
=======
                          "backgroundColor": "#FFF",
>>>>>>> origin/update/react
                          "color": "#333333",
                          "fontSize": 16,
                          "fontWeight": "400",
                          "lineHeight": 24,
                        },
                      ]
                    }
                    type="body1"
                  >
                    filled-navigation-directionleft
                  </Text>
                </View>
              </View>
            </View>
            <View
              onLayout={[Function]}
              style={null}
            >
              <View
                style={
                  Object {
                    "flex": 1,
                    "flexDirection": "row",
                  }
                }
              >
                <SvgMock
                  height={24}
                  style={
                    Object {
                      "fill": "#333333",
                    }
                  }
                  width={24}
                />
                <View
                  margin={24}
                  style={
                    Array [
                      Object {
<<<<<<< HEAD
=======
                        "flexBasis": 0,
                        "flexGrow": 1,
                        "flexShrink": 1,
>>>>>>> origin/update/react
                        "marginBottom": 24,
                        "marginLeft": 0,
                        "marginRight": 0,
                        "marginTop": 0,
                        "paddingBottom": 0,
                        "paddingLeft": 0,
                        "paddingRight": 0,
                        "paddingTop": 0,
                      },
                    ]
                  }
                >
                  <Text
                    color="#333333"
                    style={
                      Array [
                        Object {
<<<<<<< HEAD
=======
                          "backgroundColor": "#FFF",
>>>>>>> origin/update/react
                          "color": "#333333",
                          "fontSize": 16,
                          "fontWeight": "400",
                          "lineHeight": 24,
                        },
                      ]
                    }
                    type="body1"
                  >
                    filled-navigation-directionright
                  </Text>
                </View>
              </View>
            </View>
            <View
              style={
                Object {
                  "height": 0,
                }
              }
            />
          </View>
        </RCTScrollView>
      </View>
    </RCTScrollView>
  </View>
</RCTSafeAreaView>
`;
